src/components/include/application_manager/application_manager.h

```diff
@@ -93,6 +93,7 @@ class StateControllerImpl;
 struct CommandParametersPermissions;
 using policy::RPCParams;
 typedef std::vector<ApplicationSharedPtr> AppSharedPtrs;
+enum LockScreenDismissalEnabled { kNotExists = -1, kDisabled, kEnabled };
```

Comment:
@IGapchuk enum class?

Reply:
@AByzhynar Fixed in 22121736e2e4ec93cfa7a55a0e295df8f2a40223.